docs/userguide.md

```diff
@@ -8,7 +8,6 @@
     - [Deployment Image](#deployment-image)
     - [Create Clusters](#create-clusters)
   - [Helm Chart Deployment](#helm-chart-deployment)
```

Comment:
I think this title could now be `Automated Deployment` rather then `Helm Chart Deployment`, given that helm is the only mechanism of automated deployment now.

Reply:
Automated Deployment vs what - non-automated?  I think being explicit about the use of Helm is appropriate here.